hooks/command

```diff
@@ -360,6 +379,17 @@ fi
 
 command=()
 
+# Set up the copied workdir if copy-checkout
+if [[ "${BUILDKITE_PLUGIN_DOCKER_COPY_CHECKOUT:-off}" =~ ^(true|on|1)$ ]] ; then
+  COPY_COMMAND="cp -r ${workdir}_original/* ${workdir}"
+  if [[ -n "${BUILDKITE_COMMAND}" ]] ; then
+    BUILDKITE_COMMAND="${COPY_COMMAND} && ${BUILDKITE_COMMAND}"
+  fi
+  if [[ -n "${BUILDKITE_PLUGIN_DOCKER_COMMAND_0:-}" ]] ; then
+    command+=("$COPY_COMMAND && ")
```

Comment:
I haven't been able to get prepending the copy command to work correctly.  The `&&` keeps getting escaped and I'm not good enough at bash to correctly unescape them so it can execute both the copy and the user-supplied commands.  I could use some pointers here.